mix.exs

```diff
@@ -24,8 +24,8 @@ defmodule Meow.MixProject do
 
   defp deps do
     [
-      {:nx, "~> 0.3.0"},
-      {:vega_lite, "~> 0.1.1", optional: true},
+      {:nx, "0.5.0"},
+      {:vega_lite, "~> 0.1", optional: true},
```

Comment:
This one is good as is, since it's pre 1.0, 0.2 could introduce breaking changes (though it's unlikely there will be 0.2 anyway) :)

```suggestion
      {:vega_lite, "~> 0.1.1", optional: true},
```